common/views/themes/typography.ts

```diff
@@ -350,13 +350,23 @@ export const typography = css<GlobalStyleProps>`
 export function makeFontSizeClasses(): string {
   return breakpointNames
     .map(bp => {
-      return `@media (min-width: ${themeValues.sizes[bp]}px) {
-      ${Object.entries(fontSizesAtBreakpoints[bp])
-        .map(([key, value]) => {
-          return `.font-size-${key} {font-size: ${value}rem}`;
-        })
-        .join(' ')}
-    }`;
+      return `
+        @media (min-width: ${themeValues.sizes[bp]}px) {
+          ${Object.entries(fontSizesAtBreakpoints[bp])
+            .map(([key, value]) => {
+              return `.font-size-${key} {font-size: ${value}rem;}`;
+            })
+            .join(' ')}
+        }
+
+        @container card (min-width: ${themeValues.cardSizes[bp]}px) {
+          ${Object.entries(fontSizesAtBreakpoints[bp])
+            .map(([key, value]) => {
+              return `.font-size-${key} {font-size: ${value}rem !important;}`;
```

Comment:
yes, that's an `!important`. I thought/hoped `@container` would have more specificity than `@media`, but apparently not

Reply:
> There might be a way to use :has() and :not() to check if we're in a card for the @media queries, but I didn't want to get too deep into it.

We could add it as a TODO if we'd rather? I don't know how much design library changes will be affecting all things font-size?